reefperf/utils.py

```diff
@@ -7,3 +11,21 @@ def __call__(cls, *args, **kw):
         if cls.instance is None:
             cls.instance = super(Singleton, cls).__call__(*args, **kw)
         return cls.instance
+
+
+def get_node_resources(deploy_script_output):
+    node_resources = {}
+    capture_line = False
+    for line in deploy_script_output.decode('utf-8').split('\n'):
+        line = line.strip()
+        if line.lower() == CAPTURE_BEGIN:
+            capture_line = True
+            continue
+        if line.lower() == CAPTURE_END:
+            capture_line = False
+        if capture_line:
+            tokens = line.split('=')
```

Comment:
use `.partition()` for this. As it is now, if service name contains '=', it will all explode